docs/app-integration-development/09-Act-as-User-OAuth-Code-Grant.md

```diff
@@ -2,14 +2,16 @@
 tags: [app-integration-development]
 ---
 
-# Classic User OAuth: Authorization Code Grant Flow
+# Act as User Tokens via Code Grant
```

Comment:
```suggestion
# Act as User via Authorization Code Grant
```

Reply:
I changed the titles this way to try and emphasize the _type_ of token you get with each flow at the expense of complete technical detail while not making the title too long to fit in the sidebar 😰